charts/chainlink-cluster/dashboard/cmd/dashboard_deploy.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"os"
+
+	"github.com/smartcontractkit/chainlink/v2/dashboard/dashboard"
+)
+
+func main() {
+	name := os.Getenv("DASHBOARD_NAME")
+	if name == "" {
+		panic("DASHBOARD_NAME must be provided")
```

Comment:
Nit: Maybe use a fatal log or something instead? I'm not a huge fan of using panics this way, as it has a confusing layout for errors like this.